src/components/admin/campaigns/grid/CreateForm.tsx

```diff
@@ -66,7 +66,7 @@ const validationSchema: yup.SchemaOf<CampaignAdminCreateFormData> = yup
     title: yup.string().trim().min(10).max(200).required(),
     slug: yup.string().trim().min(10).max(200).optional(),
     description: yup.string().trim().min(50).max(60000).required(),
-    targetAmount: yup.number().integer().positive().required(),
+    targetAmount: yup.number().integer().positive('validation:positive-amount').required(),
```

Comment:
shouldn't we use the t() function: positive(t('...'))

Or this is done somehow automatically?

Reply:
@slavcho Yes, the translation is accessible in the validationSchema const without the t() function. All of the validation messages are displayed in this way and I kept the consistency.